.circleci/config.yml

```diff
@@ -0,0 +1,69 @@
+# Golang CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-go/ for more details
+version: 2
+jobs:
+  build:
+    docker:
+    - image: circleci/golang:1.9
+
+    #### TEMPLATE_NOTE: go expects specific checkout path representing url
+    #### expecting it in the form of
+    ####   /go/src/github.com/circleci/go-tool
+    ####   /go/src/bitbucket.org/circleci/go-tool
+    working_directory: /go/src/github.com/snowdrop/k8s-supervisor
+
+    steps:
+    # The checkout step will default to the working_directory we have already defined.
+    - checkout
+
+    - run:
+        name: Perform build
+        command: |
+          make build
+
+  release:
+    docker:
+    - image: circleci/golang:1.9
+
+    working_directory: /go/src/github.com/snowdrop/generator
```

Comment:
Why do the directory points to `/go/src/github.com/snowdrop/generator` and not `/go/src/github.com/snowdrop/k8s-supervisor` ?

Reply:
Good catch, that is bug (albeit with no actual significance). I'll fix it when I get around to updating the PR